### PR TITLE
3.1 Release Notes update

### PR DESCRIPTION
**Pull request by hardys**
Add feature and bug information, and some upgrade related notes

Co-Authored-By: Rhys Oxenham <rhys.oxenham@suse.com>
Co-Authored-By: Eduardo Mínguez <eduardo.minguez@suse.com>

Fixes: #383

### CODE REVIEW

File: asciidoc/edge-book/releasenotes.adoc [hunk 1]
```diff
@@ -30,17 +30,57 @@ NOTE: SUSE Edge z-stream releases are tightly integrated and thoroughly tested a
...
 == New Features
 
-* TBC
+* Updated to SUSE Linux Micro 6.0, Kubernetes 1.30, and Rancher Prime 2.9
+* Updated Cluster API and Metal3/Ironic versions
+* The management cluster CAPI components are now managed via Rancher Turtles
+* Management cluster upgrades are now managed via a new Upgrade Controller
+* Edge Image Builder now supports aarch64 (tech preview)
+* Stack Validation results are now published at: https://ci.edge.suse.com
+* nm-configurator updated to utilize nmstate 2.2.38
```
Comment (atanasdinov) [inline asciidoc/edge-book/releasenotes.adoc:45]:
```suggestion
* nm-configurator updated to utilize nmstate 2.2.36
```

**Review by atanasdinov — APPROVED** [review]
Perhaps we should have a more condensed version of the changes in the future, but I'm fine with this.

**Comment by hardys** [comment]
> Perhaps we should have a more condensed version of the changes in the future, but I'm fine with this.

The initial draft seemed too sparse so after discussion with @rdoxenham we added some more details, open to suggestions if we feel the list is now too verbose, but IMO it's probably OK as it is

File: asciidoc/edge-book/releasenotes.adoc [hunk 2]
```diff
@@ -30,17 +30,57 @@ NOTE: SUSE Edge z-stream releases are tightly integrated and thoroughly tested a
...
-* TBC
+* Updated to SUSE Linux Micro 6.0, Kubernetes 1.30, and Rancher Prime 2.9
+* Updated Cluster API and Metal3/Ironic versions
+* The management cluster CAPI components are now managed via Rancher Turtles
+* Management cluster upgrades are now managed via a new Upgrade Controller
+* Edge Image Builder now supports aarch64 (tech preview)
+* Stack Validation results are now published at: https://ci.edge.suse.com
+* nm-configurator updated to utilize nmstate 2.2.36
+* Edge Image Builder Enhancements:
+** Adds support for customizing SL Micro 6.0 base images (for SLE Micro 5.5 images, EIB 1.0.x must still be used)
```
Comment (e-minguez) [inline asciidoc/edge-book/releasenotes.adoc:47]: I'd say to highlight this somehow. Not sure if moving it to the known issues or not... WDYT?

**Comment by atanasdinov** [comment]
> > Perhaps we should have a more condensed version of the changes in the future, but I'm fine with this.
> 
> The initial draft seemed too sparse so after discussion with @rdoxenham we added some more details, open to suggestions if we feel the list is now too verbose, but IMO it's probably OK as it is

As agreed offline, let's merge and I'll try to come up with a proper list for the EIB ones.

File: asciidoc/edge-book/releasenotes.adoc (hunk 2 above)
Comment (hardys) [reply asciidoc/edge-book/releasenotes.adoc:47]: We discussed this on slack and it's an expected support-matrix constraint, not a known issue - given that we reference 1.1.0 everywhere in this version of the docs it should be OK, but we could mention it in the Upgrade section perhaps if you think it needs clarification?

**Comment by hardys** [comment]
> > > Perhaps we should have a more condensed version of the changes in the future, but I'm fine with this.
> > 
> > 
> > The initial draft seemed too sparse so after discussion with @rdoxenham we added some more details, open to suggestions if we feel the list is now too verbose, but IMO it's probably OK as it is
> 
> As agreed offline, let's merge and I'll try to come up with a proper list for the EIB ones.

Will do - perhaps you could also consider if the comment from @e-minguez should be addressed in that follow-up too?

**Pull request merged**